client/src/components/JobDestinationParams/JobDestinationParams.test.js

```diff
@@ -15,53 +12,50 @@ describe("JobDestinationParams/JobDestinationParams.vue", () => {
 
     const responseKeys = Object.keys(jobDestinationResponse);
 
-    let testStore, axiosMock, wrapper;
+    let testStore;
+    let wrapper;
 
     beforeEach(async () => {
-        axiosMock = new MockAdapter(axios);
         testStore = createStore();
         const propsData = {
             jobId: JOB_ID,
         };
-        axiosMock.onGet(`/api/jobs/${JOB_ID}/destination_params`).reply(200, jobDestinationResponse);
         wrapper = mount(JobDestinationParams, {
             store: testStore,
             propsData,
             localVue,
+            computed: {
+                jobDestinationParams() {
+                    return jobDestinationResponse;
+                },
+            },
         });
-        await flushPromises();
-        assert(responseKeys.length > 0, "test data is invalid!");
-    });
-
-    afterEach(() => {
-        axiosMock.restore();
+        expect(responseKeys.length > 0).toBeTruthy();
     });
 
     it("destination parameters should exist", async () => {
-        expect(Object.keys(wrapper.vm.jobDestinationParams).length).to.equals(responseKeys.length);
-        expect(wrapper.vm.jobId).to.equals(JOB_ID);
-        expect(wrapper.vm.jobDestinationParams["docker_net"]).to.equals("bridge");
-        expect(wrapper.vm.jobDestinationParams["docker_set_user"]).to.equals(null);
+        expect(Object.keys(wrapper.vm.jobDestinationParams).length).toBe(responseKeys.length);
+        expect(wrapper.vm.jobId).toBe(JOB_ID);
+        expect(wrapper.vm.jobDestinationParams["docker_net"]).toBe("bridge");
+        expect(wrapper.vm.jobDestinationParams["docker_set_user"]).toBeNull();
     });
 
     it("destination parameters should be rendered", async () => {
-        console.log(wrapper.html());
         const paramsTable = wrapper.find("#destination_parameters");
-        expect(paramsTable.isVisible()).to.equals(true);
+        expect(paramsTable.element).toBeVisible();
         const params = paramsTable.findAll("tbody > tr");
-        expect(params.length).to.equals(responseKeys.length);
+        expect(params.length).toBe(responseKeys.length);
 
         for (let counter = 0; counter < responseKeys.length - 1; counter++) {
             const parameter = params.at(counter).findAll("td");
             const parameterTitle = parameter.at(0).text();
             const parameterValue = parameter.at(1).text();
 
-            assert(responseKeys.includes(parameterTitle), "rendered parameter should exist in test data!");
+            expect(responseKeys.includes(parameterTitle)).toBeTruthy();
```

Comment:
> loosing all error messages isn't that great....
> If you want, I can try https://github.com/mattphillips/jest-expect-message on this PR.
> Otherwise, we might want to consider to define smth like this globally.
> https://stackoverflow.com/a/45375363/4870846

sorry, wrote this yesterday and did not submit. It's very weird thing on github. I doesn't write reviews right away, you kinda have to confirm afterwards 0_o . We all should move to gitlab :+1:

Reply:
We don't need the extra messages with well written tests.  Part of the point of the BDD descriptors is that when this actually fails Jest gives a very nice message that says exactly what's wrong.